PlayerInfo.h

```diff
@@ -0,0 +1,12 @@
+#ifndef TICTACTOE_SERVER_PLAYERINFO_H
+#define TICTACTOE_SERVER_PLAYERINFO_H
+
+#include <string>
+
+struct PlayerInfo {
+	std::string id;
+	std::string address;
+	bool is_active;
```

Comment:
Should this information be present here? What does it mean for a player info object to be "active"?